.gitignore

```diff
@@ -1,3 +1,9 @@
 .sysl-tmp
 bin/
-main
\ No newline at end of file
+main
+.DS_Store
+sysl.json
+output*.sysl
+.arrai
+.sysl
```

Comment:
Nit: This seems overly broad; semantically it's incorrect even if it works for this repo at this point in time. If there's a location where arrai or sysl files are generated that you don't want to commit, ignore that specifically.

Reply:
Do you disagree..?

Reply:
Yes there is an *.arrai file which needs to be checked in and petdemo.sysl is manually written to import *.yaml files.